presto-main/src/main/java/com/facebook/presto/server/ServerConfig.java

```diff
@@ -38,6 +40,30 @@ public ServerConfig setCoordinator(boolean coordinator)
         return this;
     }
 
+    public boolean isEnabledUIonSecondaryPort()
+    {
+        return enabledUIonSecondaryPort;
+    }
+
+    @Config("http-server.ui.secondary.port.enabled")
```

Comment:
I think it would be better just have that property since the default of ui.http.port will be 0 which allows OS to assign a random port to the UI's HttpServer and we cannot deduce if the ui.http.port equals to 0 means it doesn't want to enable the UI on a secondary port.